### DISCUSSION TITLE
How to disable download option in Collabora Online?

### DISCUSSION THREAD

**Discussion by psarvaiya** [CollaboraOnline/online#5959] [Q&A]
Hi team,

We created a docker image of Collabora online, following this https://collaboraonline.github.io/post/build-code/#build-code-on-ubuntu

We integrated with Nextcloud and provided the URL of Collabora online.

Our use case is to not allow any user to download files in their local system, as it will be confidential. With custom css, I can disable download option for all users in Nextcloud, and it is working as expected.

Now, if user open the writer/calc file within the document, they can still download it from Save as/Download as.

It is one of the priority requirement, that needs to be deliver, can someone please help me how can I can do this and disable the download button? 

![image](https://user-images.githubusercontent.com/112694225/223364306-355a849c-9173-47d7-89d5-73c27309bb23.png)

**Comment by kalpitrcc** [comment]
I am facing the same challenge, is there a workaround?

**Comment by TusharShelke9** [comment]
I am facing the same challenge, is there a workaround

**Comment by vmiklos** [comment]
Best to ask the same at https://forum.collaboraonline.com/, so all these questions / answers are at a single searchable place. Thanks. :-)

**Comment by zhuye0213** [comment]
modify nextcloud office plugin "Wopi.php" ,like this

	public function getHideDownload() {
		return true;
	}
